metrics/meta.go

```diff
@@ -20,8 +20,8 @@ import (
 // Metrics
 var (
 	GlobalAutoID      = "global"
-	TableAutoIDAlloc  = "table"
-	TableAutoIDRebase = "table_rebase"
+	TableAutoIDAlloc  = "id_alloc"
```

Comment:
How about just `alloc` and `rebase`?

And the subsystem name change to `autoid`?